content/en/universal_service_monitoring/_index.md

```diff
@@ -36,8 +36,9 @@ Universal Service Monitoring (USM) provides visibility into your service health
 
 Required Agent version
 : Universal Service Monitoring requires that the Datadog Agent installed alongside your containerized service be at least version 6.40 or 7.40.
+For non-containerized services see instructions [here](7)
```

Comment:
I think you meant to put this below under Prerequisites :)

Reply:
great catch
moved it